src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -1493,7 +1489,7 @@ public static boolean hasUsableWrench(EntityPlayer player, BlockPos pos)
 				return true;
 			}
 			
-			if(isCoFHHammer(tool.getItem()) && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
+			if(isCoFHHammer(tool.getItem())) // TODO Implement CoFH Hammer && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
```

Comment:
Yeah their GH is a bit weird. Like the api stuff was in CofhLib, which was a submodule of Core, and then got packaged into core, and never published as an api jar like it should have